script.js

```diff
@@ -0,0 +1,21 @@
+const hamburger = document.querySelector('.hamburger-icon');
+const mobileMenu = document.querySelector('.mobile-menu');
+const body = document.querySelector('.body');
+const close = document.querySelector('.close');
+const toolbarMobileMenu = document.querySelector('.toolbar-mobile-menu');
+
+hamburger.addEventListener('click', () => {
+  mobileMenu.classList.toggle('hidden'); // shows the mobile menu
+  // hamburger.classList.toggle('hidden'); // hides the hamburger icon
+  body.classList.toggle('overflow');
+});
+
+close.addEventListener('click', () => {
+  mobileMenu.classList.toggle('hidden');
+  body.classList.toggle('overflow');
+});
+
+toolbarMobileMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('hidden');
+  body.classList.toggle('overflow');
+});
```

Comment:
- [ ] I must admit being impressed by the fact you both almost got all the requirements right 👏 .  However, there are two slight adjustments you need to make, to have a perfect implementation.
- The margins in between the links need a little bit o space compared to that of the Figma
- the close button is not ideal in same alignment with the links

- [ ] This change might sound trivial at first glance to you, but the spacing is a major thing that gives a better user experience. a slight change can make a difference. for the close button, sticking to requirements is best, as it's the same way your clients will demand such. Kindly make the changes

## View:
![image](https://user-images.githubusercontent.com/50990925/223350503-c4830e89-734d-43bd-b073-d829c656c02d.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/223350599-ec0950fd-8cf9-484c-9a70-a52ff69dfb7d.png)